File: src/test/java/test/GetGroupAuthorizedResourcesTest.java
package test;
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class GetGroupAuthorizedResourcesTest {

    private static final String ACCESS_KEY_ID = "AUTHING_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "AUTHING_ACCESS_KEY_SECRET";

    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions();
        clientOptions.setAccessKeyId(ACCESS_KEY_ID);
        clientOptions.setAccessKeySecret(ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        GetGroupAuthorizedResourcesDto request = new GetGroupAuthorizedResourcesDto();
        request.setCode("code_2491");
        request.setNamespace("namespace_5201");
        request.setResourceType("resourceType_6652");

        AuthorizedResourceListRespDto response = client.getGroupAuthorizedResources(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/GetGroupAuthorizedResourcesTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.GetGroupAuthorizedResourcesDto{
        Code: "code_7938",
        Namespace: "namespace_0",
        ResourceType: "resourceType_8192",
    }
    response := client.GetGroupAuthorizedResources(&request)
    fmt.Println(response)
}

